flare_portal/experiments/registry.py

```diff
@@ -162,8 +162,12 @@ def register(
             class CreateMeta:
                 model = module_class
                 fields = [
-                    f.name for f in module_class._meta.fields if f.name != "sortorder"
+                    f.name
+                    for f in module_class._meta.fields
+                    if f.name != "sortorder"
+                    and (f.name not in module_class.exclude_fields)
```

Comment:
Does the conditional need to be inside a `()` pair?

Reply:
Nope, Removed.